src/test/versionbits_tests.cpp

```diff
@@ -391,6 +446,16 @@ BOOST_AUTO_TEST_CASE(versionbits_computeblockversion)
             check_computeblockversion(chainParams->GetConsensus(), static_cast<Consensus::DeploymentPos>(i));
         }
     }
+
+    {
+        // Use regtest/testdummy to ensure we always exercise the
+        // min_activation_height test, even if we're not using that in a
+        // live deployment
+        ArgsManager args;
+        args.ForceSetArg("-vbparams", "testdummy:1199145601:1230767999:403200"); // January 1, 2008 - December 31, 2008, min act height 403200
```

Comment:
dd85d5411c1702c8ae259610fe55050ba212e21e: why is the timeout set to 2008 here? In 55ac5f568a3b73d6f1ef4654617fb76e8bcbccdf  we don't the start height to `-2` here, so the test won't early-exit, but I think it's more clear to just set a far future timeout date:

```cpp
args.ForceSetArg("-vbparams", "testdummy:1199145601:4102441200:403200"); // January 1, 2008 - January 1, 2100, min act height 403200
```

Reply:
It's duplicating the old testdummy parameters. It's a fake chain so the exact times don't matter. The difference between the times can make a difference, but only if the period's short enough that two periods of 10 minute blocks will cover the entire difference.

Reply:
Maybe add an explicit test for -1/-2 as well?